misk/web/@misk/common/package.json

```diff
@@ -52,5 +53,11 @@
   "devDependencies": {
     "@misk/dev": "^0.0.25",
     "@misk/tslint": "^0.0.3"
+  },
+  "miskCachedUrls": {
+    "sq-market-font": {
+      "filepath": "sq-market.css",
+      "url": "https://d1g145x70srn7h.cloudfront.net/fonts/sqmarket/sq-market.css"
```

Comment:
we’ll probably need a better strategy for this.

Reply:
like, one problem with this is that if the resource does change, we need a mechanism to force it to get updated.

It might be simpler to check these files into Misk and make the update process completely manual.

Reply:
Agreed, the forced way would be to eventually store all of these `@misk/common` generated static assets (library code, fonts, css) on a service specific CDN with a static URL that never changes and just have the URL point at the updated assets. Then the tab code never has to change or wait for the maintainer to eventually pull the newest release from NPM.

Reply:
#444